misc-utils/fadvise.c

```diff
@@ -144,10 +153,10 @@ int main(int argc, char ** argv)
 			   offset, len,
 			   advice);
 	if (rc != 0)
-		warn(_("failed to advise"));
+		warnx(_("failed to advise: %s"), strerror(rc));
 
 	if (do_close)
 		close(fd);
 
-	return rc;
+	return !!rc;
```

Comment:
Frankly, I prefer explicit `return rc == 0 ? EXIT_SUCCESS : EXIT_FAILURE` to keep all dependence on EXIT_ macros and readable.

Reply:
Done